tests/smoke-tests/manifest.yml

```diff
@@ -6,5 +6,6 @@ applications:
       - nodejs_buildpack
     services:
       - default_aicore
+      - default_destination
```

Comment:
[q] Not sure I understand why this is needed

Reply:
Removed.